scrapers_next/me/people.py

```diff
@@ -0,0 +1,328 @@
+        p.add_link(self.source.url, note="homepage")
+
+        img = CSS("img.drop-shadow").match_one(self.root).get("src")
+        p.image = img
+
+        email = CSS("div#main-info p a").match(self.root)[0].text_content().strip()
+        p.email = email
+
+        distr_addr = CSS("div#main-info p br").match(self.root)[0].tail.strip()
+        p.district_office.address = distr_addr
+
+        try:
+            work_phone = (
+                XPath("//*[@id='main-info']/p/span[contains(text(), 'Work')]")
+                .match_one(self.root)
+                .getnext()
+            )
+            work_phone = work_phone.text_content().strip()
+            p.district_office.voice = work_phone
+        except SelectorError:
+            pass
+
+        try:
+            cell_phone = (
+                XPath("//*[@id='main-info']/p/span[contains(text(), 'Cell')]")
+                .match_one(self.root)
+                .getnext()
+            )
+            cell_phone = cell_phone.text_content().strip()
+            p.extras["Cell Phone"] = cell_phone
+        except SelectorError:
+            pass
+
+        try:
+            home_phone = (
+                XPath("//*[@id='main-info']/p/span[contains(text(), 'Home')]")
+                .match_one(self.root)
+                .getnext()
+            )
+            home_phone = home_phone.text_content().strip()
+            p.extras["Home Phone"] = home_phone
+        except SelectorError:
+            pass
+
+        seat_no = (
+            XPath("//*[@id='main-info']/p/span[contains(text(), 'Seat')]")
+            .match_one(self.root)
+            .getnext()
+        )
+        seat_no = seat_no.text_content().strip()
+        p.extras["Seat Number"] = seat_no
+
+        towns = (
+            XPath("//*[@id='main-info']/p/span[contains(text(), 'Town(s)')]")
+            .match_one(self.root)
+            .getnext()
+        )
+        towns = XPath("text()").match(towns)
+        if len(towns) > 0 and towns[0].strip() != "":
+            p.extras["towns represented"] = []
+            for town in towns:
+                p.extras["towns represented"] += [town.strip()]
+
+        cap_addr = XPath(
+            "//*[@id='welcome']/div/div[1]/div/div/table/tbody/tr[3]/td[2]/text()"
+        ).match(self.root)
+        capitol_address = ""
+        for line in cap_addr:
+            capitol_address += line.strip()
+            capitol_address += " "
+        p.capitol_office.address = capitol_address.strip()
+
+        # education
+        try:
+            occupation = (
+                XPath(
+                    "//*[@id='welcome']/div/div[1]/div/div/table/tbody/tr/td[contains(text(), 'Occupation')]"
+                )
+                .match_one(self.root)
+                .getnext()
+            )
+            p.extras["occupation"] = occupation.text_content().strip()
+        except SelectorError:
+            pass
+
+        try:
+            if (
+                XPath(
```

Comment:
I think the repeated XPath here could be replaced with 

education = Xpath(...)... 
if education is not None:
   p.extras["education"] = education...